### DISCUSSION TITLE
这两天准备转到v5, 但 test config就卡住了

### DISCUSSION THREAD

**Discussion by kingwilliam** [v2fly/v2ray-core#1924] [Q&A] [answered]
# v2rayv5 run -c 0.json

V2Ray 5.0.7 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.18.3 linux/amd64)
A unified platform for anti-censorship.

Failed to start: main/commands: failed to load config: [0.json] > infra/conf/serial: failed to read config file at line 1 char 18 > json: cannot unmarshal object into Go struct field LogConfig.log.access of type string

0.json
```
{
    "log": {
        "access": {
            "type": "none"
        }
    }
}
```
就只有这7行, 请问错在那?

**Comment by realEthanZou** [comment]
是不是 none -> None

**Reply by kingwilliam** [reply]
已測試 不是

**Comment by kingwilliam** [comment, marked as answer]
已找到原因 正确指令应该
v2rayv5 run -c 0.json -format=jsonv5

另外, 就同上面师兄说的一样
"type": 一定要 "None", 不能"none".
"level": 也一样一定要 "Info", 不能"info".
第一个一定要大草
